sdk/lib/3rdparty/libsamplerate/config.h

```diff
@@ -197,7 +197,9 @@
 #define inline __inline
 
 /* ReactOS hacks */
+#ifndef __clang__
 void __stdcall DbgBreakPoint(void);
+#endif
```

Comment:
It's not called or?

Reply:
IIRC we have it from elsewhere or it's builtin. There was an error about it here.